packit_service/service/views.py

```diff
@@ -0,0 +1,56 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+Flask views for packit-service
+"""
+from flask import Blueprint
+
+from packit_service.worker.copr_db import CoprBuildDB
+
+builds_blueprint = Blueprint("builds", __name__)
+
+
+@builds_blueprint.route(
+    "/build/<string:build_id>/<string:target>/logs", methods=("GET",)
+)
```

Comment:
What about renaming [copr-builds](https://github.com/packit-service/packit-service/blob/master/packit_service/service/api/copr_builds.py#L36) endpoint to `builds` and adding it there?

Reply:
as we discussed on a meeting: this endpoint is not an API, it's meant for users to look at

Reply:
Not sure I can agree as I don't see a difference.
Isn't API meant for users as well to look at?
Can't API endpoint return plain text / HTML or whatever your endpoint returns?

Anyway, your endpoint should at least be re-named `/build/...` -> `/builds/...`